base/uk.ac.stfc.isis.ibex.opis/.project

```diff
@@ -34,7 +34,6 @@
 	<natures>
 		<nature>org.eclipse.pde.PluginNature</nature>
 		<nature>org.eclipse.jdt.core.javanature</nature>
-		<nature>org.python.pydev.pythonNature</nature>
```

Comment:
Why has this been removed? We do have python scripts within this module (in OPIs)